app/javascript/lib/study-overview.js

```diff
@@ -0,0 +1,338 @@
+    })
+  }
+
+  const target = '#plots .panel-body'
+
+  // Duplicate calls are merely for proof-of-concept, showing we can
+  // render plots side-by-side
+  renderScatterPlot(target, rawScatter)
+
+  $('#plots').data('spinner').stop()
+
+  $('#search_annotation').val(annotation)
+  $('#gene_set_annotation').val(annotation)
+}
+
+/** Get layout object with various Plotly scatter plot display parameters */
+function getScatterPlotLayout(rawPlot) {
+  const { height, width } = calculatePlotRect()
+  let layout = getBaseLayout(height, width)
+
+  let dimensionProps
+  if (rawPlot.is3D) {
+    const camera = $('#cluster-plot').data('camera')
+    dimensionProps = get3DScatterProps(camera, window.SCP.cluster)
+  } else {
+    dimensionProps = get2DScatterProps(window.SCP.cluster)
+  }
+
+  layout = Object.assign(layout, dimensionProps)
+
+  return layout
+}
+
+function attachEventHandlers() {
+  // For inferCNV ideogram
+  $('#ideogram_annotation').on('change', function() {
+    const ideogramFiles = window.SCP.study.ideogramFiles
+    const fileId = $(this).val() // eslint-disable-line
+    if (fileId !== '') {
+      const ideogramAnnot = ideogramFiles[fileId]
+      window.ideogramInferCnvSettings = ideogramAnnot.ideogram_settings
+      window.initializeIdeogram(ideogramAnnot.ideogram_settings.annotationsPath)
+    } else {
+      $('#tracks-to-display, #_ideogramOuterWrap').html('')
+      $('#ideogramTitle').remove()
+    }
+  })
+
+  // resize listener
+  $(window).on('resizeEnd', () => {resizePlots()})
+
+  // listener for cluster nav, specific to study page
+  $('#annotation').change(function() {
+    $('#cluster-plot').data('rendered', false)
+    const an = $(this).val() // eslint-disable-line
+    // keep track for search purposes
+    $('#search_annotation').val(an)
+    $('#gene_set_annotation').val(an)
+    drawScatterPlot()
+  })
+
+  $('#subsample').change(function() {
+    $('#cluster-plot').data('rendered', false)
+    const subsample = $(this).val() // eslint-disable-line
+    $('#search_subsample').val(subsample)
+    $('#gene_set_subsample').val(subsample)
+    drawScatterPlot()
+  })
+
+  $('#cluster').change(function() {
+    $('#cluster-plot').data('rendered', false)
+    const newCluster = $(this).val() // eslint-disable-line
+    // keep track for search purposes
+    $('#search_cluster').val(newCluster)
+    $('#gene_set_cluster').val(newCluster)
+    drawScatterPlot()
+  })
+}
+
+/** Initialize the "Explore" tab in Study Overview */
+export default async function initializeExplore() {
+  window.SCP.study = {}
+  window.SCP.plots = []
+  window.SCP.plotRects = []
+
+  window.SCP.startPendingEvent('user-action:page:view:site-study',
+    { speciesList: window.SCP.taxons },
+    'plot:',
+    true)
+
+  $('#cluster-plot').data('rendered', false)
+
+  const baseCamera = {
+    'up': { 'x': 0, 'y': 0, 'z': 1 },
+    'center': { 'x': 0, 'y': 0, 'z': 0 },
+    'eye': { 'x': 1.25, 'y': 1.25, 'z': 1.25 }
+  }
```

Comment:
baseCamera feels like it should be a constant declared outside of this function?

Reply:
Quite likely -- I added a TODO in b24e07f so this can be refactored as other views (e.g. single-gene) get implemented.